diff --git a/Makefile.linux b/Makefile.linux
index d83af66..15f605a 100644
--- a/Makefile.linux
+++ b/Makefile.linux
@@ -46,18 +46,18 @@ endif
 # needs :
 #			https://github.com/m5evt/unixcw-3.5.1.git
 
-CWDAEMON_INCLUDE=CWDAEMON
+#CWDAEMON_INCLUDE=CWDAEMON
 
-ifeq ($(CWDAEMON_INCLUDE),CWDAEMON)
-CWDAEMON_OPTIONS=-D CWDAEMON
-CWDAEMON_LIBS=-lcw
-CWDAEMON_SOURCES= \
-cwdaemon.c
-CWDAEMON_HEADERS= \
-cwdaemon.h
-CWDAEMON_OBJS= \
-cwdaemon.o
-endif
+#ifeq ($(CWDAEMON_INCLUDE),CWDAEMON)
+#CWDAEMON_OPTIONS=-D CWDAEMON
+#CWDAEMON_LIBS=-lcw
+#CWDAEMON_SOURCES= \
+#cwdaemon.c
+#CWDAEMON_HEADERS= \
+#cwdaemon.h
+#CWDAEMON_OBJS= \
+#cwdaemon.o
+#endif
 
 # MIDI code from piHPSDR written by Christoph van Wullen, DL1YCF.
 MIDI_INCLUDE=MIDI
@@ -261,12 +261,15 @@ clean:
 	-rm -f $(PROGRAM)
 
 install: $(PROGRAM)
-	cp $(PROGRAM) /usr/local/bin
-	if [ ! -d /usr/share/linhpsdr ]; then mkdir /usr/share/linhpsdr; fi
-	cp hpsdr.png /usr/share/linhpsdr
-	cp hpsdr_icon.png /usr/share/linhpsdr
-	cp hpsdr_small.png /usr/share/linhpsdr
-	cp linhpsdr.desktop /usr/share/applications
+	mkdir -p $(PREFIX)/bin
+	mkdir -p $(PREFIX)/share/icons/hicolor/512x512/apps
+	mkdir -p $(PREFIX)/share/applications
+	cp $(PROGRAM) $(PREFIX)/bin
+	if [ ! -d $(PREFIX)/share/linhpsdr ]; then mkdir $(PREFIX)/share/linhpsdr; fi
+	cp hpsdr.png $(PREFIX)/share/linhpsdr
+	cp hpsdr_icon.png $(PREFIX)/share/icons/hicolor/512x512/apps/io.github.g0orx.linhpsdr.png
+	cp hpsdr_small.png $(PREFIX)/share/linhpsdr
+	cp linhpsdr.desktop $(PREFIX)/share/applications/io.github.g0orx.linhpsdr.desktop
 
 debian:
 	cp $(PROGRAM) pkg/linhpsdr/usr/local/bin
diff --git a/linhpsdr.desktop b/linhpsdr.desktop
index a2c1162..9239c7d 100644
--- a/linhpsdr.desktop
+++ b/linhpsdr.desktop
@@ -1,9 +1,9 @@
 [Desktop Entry]
-Version=0.1
+Version=1.0
 Type=Application
 Terminal=false
-Name[eb_GB]=linHPSDR
+Name=LinHPSDR
+Comment=Linux HPSDR
 Exec=linhpsdr
-Icon=/usr/share/linhpsdr/hpsdr_small.png
-Name=linHPSDR
-
+Icon=io.github.g0orx.linhpsdr
+Categories=AudioVideo;Audio;Network;HamRadio;
diff --git a/main.c b/main.c
index 2d36744..33e4d41 100644
--- a/main.c
+++ b/main.c
@@ -501,7 +501,7 @@ static void activate_hpsdr(GtkApplication *app, gpointer data) {
   g_print("opengl: %d\n",opengl);
 
 //#ifdef __APPLE__
-  sprintf(png_path,"/usr/local/share/linhpsdr/hpsdr.png");
+  sprintf(png_path,"/app/share/linhpsdr/hpsdr.png");
 //#else
 //  sprintf(png_path,"/usr/share/linhpsdr/hpsdr.png");
 //#endif
